README.rst

```diff
@@ -16,22 +16,53 @@ mflux-ai
 
 
 
-Open source code for the mflux-ai python package
+Official the mflux-ai python library.
+
 
 
 * Free software: BSD license
 * Documentation: https://mflux-ai.readthedocs.io.
 
-
 Features
 --------
+- acquire and set environment variables required by mflux-server
+- cache the environment variables
+
+
+Quickstart
+----------
+Install mflux-ai
+
+    pip install mflux-ai
+
+Import mflux_ai
+
+    Import  mflux-ai
+
+    mflux_ai.set_env_vars(token="9s9kQ0D86wWKUHdPMj0HHA", server_host"http://www.your-sever-name.mflux.ai")
```

Comment:
Remove the server_host parameter in this example. It should normally not be used.